package.json

```
@@ -3,14 +3,16 @@
   "version": "1.0.0",
   "description": "reduced.to is a web application that reduces the length of link URL. So it's easier to remember, share and track.",
   "scripts": {
-    "test:all": "npm run test:backend",
+    "test:all": "npm run test:backend && npm run test:e2e",
     "test:backend": "npm --prefix ./apps/backend run test",
+    "test:e2e": "npm --prefix ./e2e run test",
     "build:all": "npm run build:frontend && npm run build:backend",
     "build:frontend": "npm --prefix ./apps/frontend run build && npm --prefix ./apps/frontend run build.server",
     "build:backend": "npm --prefix ./apps/backend run build",
-    "install:all": "npm run install:frontend && npm run install:backend",
+    "install:all": "npm run install:frontend && npm run install:backend && npm run install:e2e",
     "install:frontend": "cd ./apps/frontend && npm i",
     "install:backend": "cd ./apps/backend && npm i",
+    "install:e2e": "cd ./e2e && npm ci && npx playwright install --with-deps",
```

Why are we running clean install instead of install?
```suggestion
"install:e2e": "cd ./e2e && npm i && npx playwright install --with-deps",
```